ports/gstplugin/servowebsrc.rs

```diff
@@ -663,10 +729,20 @@ impl BaseSrcImpl for ServoWebSrc {
             gl_memory,
             result,
         };
+
         let data = &mut task as *mut RunOnGLThread as *mut c_void;
         unsafe { gst_gl_context_thread_add(gl_memory.mem.context, Some(fill_on_gl_thread), data) };
         task.result?;
 
+        // Put down a GL sync point if needed
+        let gst_buffer = buffer.to_glib_none();
+        let sync_meta_type = unsafe { gst_gl_sync_meta_api_get_type() };
+        let sync_meta =
+            unsafe { gst_buffer_get_meta(gst_buffer.0, sync_meta_type) } as *mut GstGLSyncMeta;
+        if !sync_meta.is_null() {
+            unsafe { gst_gl_sync_meta_set_sync_point(sync_meta, gl_memory.mem.context) };
```

Comment:
We should be able to use:
* https://slomo.pages.freedesktop.org/rustdocs/gstreamer/gstreamer/buffer/struct.BufferRef.html#method.get_meta
* https://slomo.pages.freedesktop.org/rustdocs/gstreamer/gstreamer_gl/struct.GLSyncMeta.html
* https://slomo.pages.freedesktop.org/rustdocs/gstreamer/gstreamer_gl/struct.GLSyncMeta.html#method.set_sync_point

Reply:
Done.